TAF/utils/src/setup/edgex.py

```diff
@@ -13,6 +13,12 @@
 import TAF.utils.src.setup.startup_checker as checker
```

Comment:
should these be under TUC?? seems to be generic enough that all the TAF projects might want to use in future.

Reply:
Also requesting the WG to review the subfolders under TAF->Utils (setup, verify, report etc) and see which ones does and doesn't make sense to be there..

Reply:
It's not stable to move these files to common repository. This PR just enough to make these tests work and deliver the report, so I consider keeping these files in the TAF/Utils.

Reply:
can we create a separate PR for revisitng the Utils and possible porting of generic libraries to TUC?

Reply:
Yes, you can create a separate PR to refactor the TAF/Utils to edgex-taf-common/TUC if you have another repository need to use these functions.

Reply:
Noted. thanks!